app/controllers/events_controller.rb

```diff
@@ -14,6 +15,7 @@ def upcoming
 
   def create
     @event = Event.new(event_params)
+    authorize @event
```

Comment:
@graygilmore I think this could be interesting to do. But given we will be loading different resources for different actions, we would need to write something like what Cancan provides for 'intelligently' loading the resource based on the action the controller is performing. 

I think for now being explicit is ok